config/templates/template.clusterserviceversion.yaml

```diff
@@ -0,0 +1,81 @@
+    mediatype: image/png
+  description: >
+    Composable is an overlay operator that can wrap any resource (native Kubernetes or CRD instance) and allows it to be dynamically configurable. 
+    Any field of the underlying resource can be specified with a reference to a secret, a configmap, or any field of another Kubernetes object. 
+    It enables the complete declarative executable specification of collections of inter-dependent resources.
+    
+    The Composable Operator is currently in preview. It will get updated
+    as we release new versions of the [upstream repository](https://github.com/IBM/composable).
+
+    ## Requirements
+
+    The operator can be installed on any OLM-enabled Kubernetes cluster with version >= 1.11. 
+    
+    ## Using the Composable Operator
+
+    For additional configuration options, samples and more information on using the operator, consult 
+    the [IBM Cloud Operator documentation](https://github.com/IBM/cloud-operators).
```

Comment:
Should point to the composable doc not cloud operators docs

Reply:
done